README.md

```diff
@@ -152,6 +152,85 @@ $
 $ docker run -p 3000:3000 -p 35729:35729 -v /Users/mdl/workspace/mean-stack/mean/public:/home/mean/public -v /Users/mdl/workspace/mean-stack/mean/app:/home/mean/app --link db:db_1 mean
 ```
 
+## Deploying To Cloud Foundry
```

Comment:
@codydaig is it best to create a separate pull request for that branch with the documentation in it?